estate/models/res_users.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime, timedelta
+from datetime import date
+from odoo import fields, models
+from odoo import api
+from odoo import exceptions
+from odoo.exceptions import ValidationError
```

Comment:
```suggestion
from odoo import fields, models
```
mejor solo importar lo necesario